amr-wind/equation_systems/vof/volume_fractions.H

```diff
@@ -1,34 +1,249 @@
-#ifndef VolumeFraction_H_
-#define VolumeFraction_H_
-
-#include <cmath>
+#ifndef VOLUME_FRACTIONS_H_
+#define VOLUME_FRACTIONS_H_
 
 #include <AMReX_FArrayBox.H>
+#include <cmath>
 
 namespace amr_wind {
 namespace multiphase {
 
+/**---------------------------------------------------------------
+// FD32 - Youngs Finite Difference Gradient Scheme
+// the gradient is computed with a multiplicative factor of -32:
+//  mm = - 32 * grad (c)
+//----------------------------------------------------------------
+//
+// Known problems: the index (1,1,1), i.e. the central cell in the block, never
+occurs:
+// Therefore an isolated droplet will have a normal with all components to zero.
+// Translated into f90 by Stephane Z.
+*/
+
```

Comment:
Prefer to use doxygen style comment here. 

```suggestion
/** Young's finite-difference gradient scheme.
 *
 *  The gradient is computed with a multiplicative factor of -32:
 *  \f[ mm = - 32 * \grad (c) \f]
 *
 *  Known problems: the index (1,1,1), i.e. the central cell in the block, never
 *  occurs: Therefore an isolated droplet will have a normal with all components
 *  to zero.
 */
```